src/main/java/build/buildfarm/worker/shard/Worker.java

```diff
@@ -786,7 +786,7 @@ public void run() {
 
   public void start() throws InterruptedException {
     try {
-      backplane.start();
+      backplane.start(config.getPublicName());
```

Comment:
Not related to this diff, but I kind of wish we saved the entire config in redis so we could query anything about a worker/scheduler.  
key: ip / value: config

Reply:
Sounds like a new story for the backlog!